c7n/resources/glue.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2016-2017 Capital One Services, LLC
```

Comment:
for new files, copyright year just read the date its written, intent is that years reflect dates modified.